tests/e2e/features/presentationViewer.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Markdown presentation viewer
+  As a user
+  I want to view the markdown in the presentation format
+  So that I can view the markdown documents in the presentation format
```

Comment:
Please make a clear distinction between **what** the user wants to do (`I want to ...`) and **why** the user wants to do this (`so that ...`).  
It might be helpful to have a look at the user story that was initially defined for this extension app in [this issue](https://github.com/JankariTech/web-app-presentation-viewer/issues/1)